Js/index.js

```diff
@@ -4,24 +4,172 @@ let btnCancelMobileMenu = document.querySelector("#btn-close-nav");
 const PortfolioClicked = document.querySelector(".link-portfolio");
 const AboutClicked = document.querySelector(".link-about");
 const ContactClicked = document.querySelector(".link-contact");
+const allBtnPopups = document.querySelectorAll(".btn-pop");
```

Comment:
- [ ] nice adding the close button as the previous reviewer asked but the button is not well-positioned  it should be the right side  please try to fix that 
I encourage you to give it a `position: absolute ` so you can make fixes in one place

- [ ]  Yours Design 
![image](https://user-images.githubusercontent.com/61790587/167960641-ee755898-b2bb-4c38-bb3a-95b11164bcd8.png)